narwhals/_arrow/expr.py

```diff
@@ -194,6 +194,9 @@ def sum(self) -> Self:
     def drop_nulls(self) -> Self:
         return reuse_series_implementation(self, "drop_nulls")
 
+    def series_shift(self, n: int) -> Self:
+        return reuse_series_implementation(self, "series_shift", n)
```

Comment:
```suggestion
    def shift(self, n: int) -> Self:
        return reuse_series_implementation(self, "shift", n)
```

these should have the same name otherwise `narwhals` will not be able to find it